tasks/agent.py

```diff
@@ -412,6 +412,10 @@ def omnibus_build(ctx, puppy=False, agent_binaries=False, log_level="info", base
                 env['SIGN_PFX'] = "{}".format(pfxfile)
                 env['SIGN_PFX_PW'] = "{}".format(pfxpass)
 
+            if sys.platform == 'darwin':
+                # Target MacOS 10.12
+                env['MACOSX_DEPLOYMENT_TARGET'] = '10.12'
```

Comment:
We could maybe define an optional argument to the build command that would default to `10.12` so that we could override the target easily from the CLI? This looks good though, just figured it could be a nice knob.

Reply:
I thought about it too, we can add that later.